apps/wizard/utils/io.py

```diff
@@ -75,13 +75,21 @@ def get_changed_grapher_steps(files_changed: Dict[str, Dict[str, str]]) -> List[
 
         # Identify grapher data steps, and ignore the rest.
         if file_path.startswith(STEP_DIR.relative_to(BASE_DIR).as_posix()) and file_path.endswith(".py"):
```

Comment:
(Please change the comment above, which is no longer accurate).
Wouldn't we want to catch snapshot changes as well? Maybe:
```
if file_path.startswith((STEP_DIR.relative_to(BASE_DIR).as_posix(), SNAPSHOTS_DIR.relative_to(BASE_DIR).as_posix())) and file_path.endswith((".py", ".dvc")):
```